python/neutron-understack/neutron_understack/nautobot.py

```diff
@@ -26,6 +30,30 @@ class NautobotCustomFieldNotFoundError(exc.NeutronException):
     message = "Custom field with name %(cf_name)s not found for %(obj)s"
 
 
+@dataclass
+class VlanPayload:
+    id: str
+    network_id: str
+    network_name: str
+    vid: int
+    vlan_group_name: str
+    status: str = "Active"
+
+    def to_dict(self) -> dict:
+        return {
+            "id": self.id,
+            "vid": self.vid,
+            "name": utils.derive_from_network(
+                network_id=self.network_id, network_name=self.network_name
+            ),
+            "vlan_group": {"name": self.vlan_group_name},
+            "status": {"name": self.status},
```

Comment:
```suggestion
            "status": self.status,
```

They look up `name` and `id` the same way and flatten the dictionary internally in Django REST framework so there's no advantage here to that.